lib/matplotlib/backends/backend_ps.py

```diff
@@ -764,10 +764,29 @@ def print_ps(self, outfile, *args, **kwargs):
     def print_eps(self, outfile, *args, **kwargs):
         return self._print_ps(outfile, 'eps', *args, **kwargs)
 
-    def _print_ps(self, outfile, format, *args,
-                  papertype=None, dpi=72, facecolor='w', edgecolor='w',
-                  orientation='portrait',
-                  **kwargs):
+    def _print_ps(
+            self, outfile, format, *args,
+            dpi=72, metadata=None, papertype=None, orientation='portrait',
+            **kwargs):
+
+        self.figure.set_dpi(72)  # Override the dpi kwarg
+
+        dsc_comments = {}
+        if isinstance(outfile, (str, os.PathLike)):
+            dsc_comments["Title"] = \
+                os.fspath(outfile).encode("ascii", "replace").decode("ascii")
+        dsc_comments["Creator"] = (metadata or {}).get(
+            "Creator",
+            f"matplotlib version {mpl.__version__}, http://matplotlib.org/")
+        source_date_epoch = os.getenv("SOURCE_DATE_EPOCH")
```

Comment:
I would still keep the reproducible builds link.

Reply:
put it back